all/generate/.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -9,10 +9,10 @@ assignees: ''
 
 ## Bug Report Checklist
 
-- [ ] I have checked the existing [documentation](https://github.com/finbourne/lusid-sdk-python-preview/wiki) for this SDK
+- [ ] I have checked the existing [documentation](../../../../wiki) for this SDK
```

Comment:
I don't think relative links work, can probably check by editing one in the main sdk repo to test

Reply:
You don't think I'd have put this up without testing it do you?